core/kazoo_schemas/src/kz_json_schema_extensions.erl

```diff
@@ -115,3 +116,38 @@ validate_attachment_oauth_doc_id(Value, State) ->
             lager:debug("~s", [ErrorMsg]),
             jesse_error:handle_data_invalid('external_error', ErrorMsg, State)
     end.
+
+maybe_check_params_stability_level(Value, Schema, State) ->
+    case kz_json:get_value(<<"properties">>, Schema) of
+        'undefined' -> State; %% Skipping schema with out `properties' property
+        Properties -> check_params_stability_level(Properties, Value, State)
+    end.
+
+check_params_stability_level(Properties, Value, State) ->
+    {ok, CBConfig} = kz_datamgr:open_cache_doc(<<"system_config">>, <<"crossbar">>),
+    SystemSL = kz_json:get_binary_value([<<"default">>, <<"stability_level">>], CBConfig),
```

Comment:
To get a value from system_config you have to use `kapps_config` module.

So instead of this line and above line use this:
```
kapps_config:get_ne_binary_value(<<"crossbar">>, <<"stability_level">>, <<"stable">>)
```
It gets `stability_level` key from crossbar category (a document in system_config db), if it's not found it set/return the default value `stable`.

After this change make sure to compile and run `make apis` to update crossbar system_config schema.